diesel.toml

```diff
@@ -0,0 +1,2 @@
+[print_schema]
+file = "crates/db_schema/src/schema.rs"
```

Comment:
Maybe it would be useful to add a CI step which generates the schema to check that its up to date.

Reply:
Done